packages/flutter/lib/src/rendering/custom_paint.dart

```diff
@@ -227,7 +227,7 @@ abstract class CustomPainter extends Listenable {
   /// that in most cases semantics change when something new is drawn.
   bool shouldRebuildSemantics(covariant CustomPainter oldDelegate) => shouldRepaint(oldDelegate);
 
-  /// Called whenever a new instance of the custom painter delegate class is
+  /// Called whenever a new instance of the custom clipper delegate class is
```

Comment:
Why should this be changed to clipper? This is in the CustomPainter class and painter delegate seems correct here?

Reply:
@goderbauer  Okay, I am reverting it can you suggest how can I improve this [#60447](https://github.com/flutter/flutter/issues/60447)

Reply:
#60447 is talking about the docs on CustomClipper, not CustomPainter.